packages/bootleg-app/src/TraderComponent.js

```diff
@@ -1,82 +1,137 @@
 import React from 'react';
 import { drizzleConnect } from 'drizzle-react';
-import { ContractData, ContractForm } from 'drizzle-react-components';
-import { Button, Form, Input } from 'rimble-ui';
+import {
+  AccountData,
+  ContractData,
+  ContractForm
+} from 'drizzle-react-components';
+import { Button, Form, Input, Card } from 'rimble-ui';
+
+const tokenPurchasePrice = 1000000;
 
 const TraderComponent = ({ accounts, drizzleStatus }) => (
   <div className="App">
-    <div>
-      <h1>Drizzle Examples</h1>
-      <p>Examples of how to get started with Drizzle in various situations.</p>
-    </div>
-    <div className="section">
-      <h2>The Token</h2>
-      <ContractData contract="BootlegTraderApp" method="bootlegTokenId" />
-      <h2>Token URI</h2>
-      <ContractData contract="BootlegTraderApp" method="getTokenURI" />
-      <h2>Token Owner</h2>
-      <ContractData contract="BootlegTraderApp" method="getOwner" />
-      <h2>Your Balance</h2>
-      <ContractData contract="BootlegTraderApp" method="getBalance" />
-      <h2>Active Account</h2>
-      {JSON.stringify(accounts)}
-      <h2>Purchase</h2>
-      <ContractForm
-        contract="BootlegTraderApp"
-        method="purchase"
-        render={({
-          inputs,
-          inputTypes,
-          state,
-          handleInputChange,
-          handleSubmit
-        }) => (
-          <Form onSubmit={handleSubmit}>
-            <Button
-              key="submit"
-              type="button"
-              onClick={handleSubmit}
-              size={'medium'}
-            >
-              Purchase
-            </Button>
-          </Form>
-        )}
-      />
-      <h2>List for sale</h2>
-      <ContractForm
-        contract="BootlegTraderApp"
-        method="setTokenPrice"
-        render={({
-          inputs,
-          inputTypes,
-          state,
-          handleInputChange,
-          handleSubmit
-        }) => (
-          <Form onSubmit={handleSubmit}>
-            {inputs.map((input, index) => (
-              <Input
-                style={{ fontSize: 30 }}
-                key={input.name}
-                type={inputTypes[index]}
-                name={input.name}
-                value={state[input.name]}
-                placeholder={input.name}
-                onChange={handleInputChange}
-              />
-            ))}
-            <Button
-              key="submit"
-              type="button"
-              onClick={handleSubmit}
-              size={'medium'}
-            >
-              List for sale
-            </Button>
-          </Form>
+    <Card fontSize={4}>
+      <span>You are logged in with the following account:</span>
+      <AccountData
+        accountIndex="0"
+        units="ether"
+        precision="3"
+        render={({ address, balance, units }) => (
+          <div>
```

Comment:
to avoid the div spaghetti you could replace `<div>` with `<>` which is shorthand for a `React.fragment` which simply just renders as nothing but still gives react a single node to work with.